contracts/VotingMachines/IntVoteInterface.sol

```diff
@@ -9,6 +9,12 @@ interface IntVoteInterface {
     modifier onlyProposalOwner(bytes32 _proposalId) {revert(); _;}
     modifier votable(bytes32 _proposalId) {revert(); _;}
 
+    event NewProposal(bytes32 indexed _proposalId, address indexed _avatar, uint _numOfChoices, address _proposer, bytes32 _paramsHash);
```

Comment:
please remove this from the interface.

Reply:
Can you please explain why, for the record?

Reply:
if it is the intrerface so remove that from the GenesisProtocol and AbsoluteVote

Reply:
Done

Reply:
AbsoluteVote adds a parameter, so I left it in.  Same deal for `GP.ExecuteProposal`.  (See my comments in the PR description.)

Reply:
so remove it from the interface

Reply:
To many unneeded events.

Reply:
OK, then I would vote to eliminate `AbsoluteVoteProposal` (the existing event in AV) if it is unneeded.  The `IntVoteProposal` version *is* needed.

Similarly with `ExecuteProposal` in GP.

We need to be able to subscribe to these events without knowing what the voting machine is.  You OK with that?

Reply:
AbsoluteVoteProposal does not exist in ARC.
you mean VoteProposal ?

Reply:
I describe what I mean by "AbsoluteVoteProposal" in my comment in this thread just above.

Reply:
But yes, I would propose to eliminate the existing `AbsoluteVote.VoteProposal` and `GenesisProtocol.ExecuteProposal` events if you don't like my proposal above to rename and emit them as separate events.

Reply:
@orenyodfat 

This is a long and confusing thread, and it needs to be resolved, so let me summarize where we are at on it:

There is a mismatch in two different events between IntVoteInterface and GenesisProtocol and AbsoluteVote:

* GP.ExecuteProposal has an extra param: ExecutionState _executionState
* AV.VoteProposal has an extra param: bool isOwnerVote param

In order to conform to the IntVoteInterface, AV and GP needs to emit the exact events as they are defined in the interface.  As far as I know, there are two ways to do this:

1) Remove the extra parameters from the extended events in AV and GP
2) Have AV and GP throw both the interface version and extended versions of the events.  We would rename the extended events.

Can you please indicate which is the preferred option, thanks?

Reply:
Decision is to add new events: AVVoteProposal and GPExecuteProposal.  This is done.